src/resolvers/User.js

```diff
@@ -0,0 +1,279 @@
+import {
+  ApolloError,
+  AuthenticationError,
+  UserInputError,
+} from "apollo-server-express";
+import validator from "email-validator";
```

Comment:
can rename the validator?
too much validate here I confuse
**not mandatory la**

Reply:
ok